src/test/java/nextstep/favorite/acceptance/FavoriteAcceptanceTest.java

```diff
@@ -1,8 +1,175 @@
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 @DisplayName("즐겨찾기 관련 기능")
 public class FavoriteAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    private DatabaseCleanup databaseCleanup;
+
+    long 강남역, 역삼역, 선릉역, 이호선;
+    String accessToken;
+
+    @BeforeEach
+    public void setUp() {
+        databaseCleanup.execute();
+
+        강남역 = 지하철역_생성("강남역").jsonPath().getLong("id");
+        역삼역 = 지하철역_생성("역삼역").jsonPath().getLong("id");
+        선릉역 = 지하철역_생성("선릉역").jsonPath().getLong("id");
+
+        이호선 = 지하철_노선_생성(new LineRequest("2호선", "green", 강남역, 역삼역, 10)).jsonPath().getLong("id");
+        지하철_구간_추가(이호선, 역삼역, 선릉역, 10);
+
+        MemberFixture memberFixture = new MemberFixture();
+        accessToken = memberFixture.getAccessToken();
+    }
+
+    @Test
+    void 즐겨찾기_생성() {
+        // give
+
+        // when
+        ExtractableResponse<Response> response = 즐겨찾기_생성_요청(강남역, 선릉역, accessToken);
+
+        // then
+        FavoriteResponse favoriteResponse = response.as(FavoriteResponse.class);
+
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
+                () -> assertThat(favoriteResponse.getSource().getId()).isEqualTo(강남역),
+                () -> assertThat(favoriteResponse.getTarget().getId()).isEqualTo(선릉역)
+        );
+    }
+
+    @Test
+    void 즐겨찾기_조회() {
+        // give
+        즐겨찾기_생성_요청(강남역, 선릉역, accessToken);
+
+        // when
+        ExtractableResponse<Response> response = 즐겨찾기_조회_요청(accessToken);
+
+        List<FavoriteResponse> favorites = response.as(new TypeRef<>() {});
+
+        // then
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                () -> assertThat(favorites).hasSize(1),
+                () -> assertThat(favorites.stream().anyMatch(
+                        favorite -> favorite.getSource().getId().equals(강남역)
+                        && favorite.getTarget().getId().equals(선릉역))).isTrue()
+        );
+    }
+
+    @Test
+    void 즐겨찾기_삭제() {
+        // give
+        ExtractableResponse<Response> response = 즐겨찾기_생성_요청(강남역, 선릉역, accessToken);
+        long favoriteId = response.jsonPath().getLong("id");
+
+        // when
+        즐겨찾기_삭제_요청(accessToken, favoriteId);
+
+        // then
+        List<FavoriteResponse> favorites = 즐겨찾기_조회_요청(accessToken).as(new TypeRef<>() {});
+
+        assertAll(
+                () -> assertThat(favorites).hasSize(0),
+                () -> assertThat(favorites.stream().anyMatch(
+                        favorite -> favorite.getSource().getId().equals(강남역)
+                                && favorite.getTarget().getId().equals(선릉역))).isFalse()
+        );
+    }
+
+    @DisplayName("로그인 하지않으면 즐겨찾기 생성 불가")
+    @Test
+    void 로그인X_즐겨찾기_생성() {
+        // give
+
+        // when
+        ExtractableResponse<Response> response = 즐겨찾기_생성_요청(강남역, 선릉역, "");
```

Comment:
로그인 하지 않은 상황을 쉽게 알 수 있게 표현하면 어떨까요?
```suggestion
        // give
        String 비로그인_토큰 = "";

        // when
        ExtractableResponse<Response> response = 즐겨찾기_생성_요청(강남역, 선릉역, 비로그인_토큰);
```